Created 2016-10-24-configuring-model-driven-telemetry-mdt-with-ydk-and-xr-native-yang-model.md

marumer opened the pull request:
This pull request has been automatically generated by prose.io.

marumer committed:
Created 2016-10-24-configuring-model-driven-telemetry-mdt-with-ydk-and-xr-native-yang-model.md

scadora commented:
Thanks for doing this, Marco!  The more examples we have, the better.  Some thoughts below.

Who is your intended audience?  If the audience is external, then it will be important to do this with a recently released or about to be released version (so 6.1.1 or 6.1.2/3).  6.2.1 is a ways out there, so we'd have to hide it for a while for that version.  

Also suggest editing for an external audience -- remove comments about what the engineering team is working on, the preferred method for PoCs, etc, and try to just focus on what works and how.  Basically, the tutorial gives an example of how to configure TCP dial-out using the native model and YDK.  That's all you have to say :)

I noticed your sensor-path specifies the path out to a leaf (bytes-received) -- since we only retrieve to the container above that, I'd just terminate the path at "generic-counters"

marumer committed:
Update 2016-10-24-configuring-model-driven-telemetry-mdt-with-ydk-and-xr-native-yang-model.md

Updating for external consumption - removing references to Openconfig YANG gaps.
Fixing the sensor path removing the typo more specific byte-received path

scadora reviewed (approved):
Hi, Marco.
Thanks for the updates.  Looks good.  I'm going to approve, but could you say more about the use of the Empty type in the subscription?  What happens if you don't include that line?  From the CLI's perspective, you don't have to do anything special to activate the subscription, so I'm wondering what this actually does.
